### PR TITLE
Set cache and security headers

### PR DESCRIPTION
**Pull request by pmac**

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -3,6 +3,14 @@
   command = "eleventy"
   functions = "./functions"
 
+[[headers]]
+  for = "/*"
+  [headers.values]
+    X-Content-Type-Options = "nosniff"
+    X-Frame-Options = "DENY"
+    X-XSS-Protection = "1; mode=block"
+    Cache-Control = "max-age=600, public"
```
Comment (sciurus) [inline netlify.toml:12]: I'm wondering if this is right based on https://docs.netlify.com/routing/headers/#multi-value-headers , they use document using multiline strings there.


```
Cache-Control = '''
max-age=600,
public'''
```

Comment (pmac) [reply netlify.toml:12]: I saw that, but I figured with only a short need here it'd be fine on one line like they did with the XSS-Protection one.

Comment (pmac) [reply netlify.toml:12]: I took it to mean you can use multiline rather than you should... but ¯\_(ツ)_/¯

Comment (sciurus) [reply netlify.toml:12]: Yeah, that make more sense than a strict interpretation.

**Comment by sciurus** [comment]
LGTM, a couple tests from the preview site


```
$ curl -v https://606661315a6f0d0008e4dbda--pocket-static-staging.netlify.app/en/pocket-and-firefox/ 2>&1 > /dev/null | grep cache-control
< cache-control: max-age=600,public
```

```
$ curl -v https://606661315a6f0d0008e4dbda--pocket-static-staging.netlify.app/en/pocket-and-firefox/ 2>&1 > /dev/null | grep cache-control
curl -v https://606661315a6f0d0008e4dbda--pocket-static-staging.netlify.app/marketing-assets/web-discover/_next/static/chunks/8478bddf1c0e08e495aab196b1b9af0b03f1c644.7a39688da8a9f7fcbbf4.js 2>&1 > /dev/null | grep cache-control
< cache-control: max-age=600,public
```

```
curl -v https://606661315a6f0d0008e4dbda--pocket-static-staging.netlify.app/marketing-assets/web-discover/_next/static/images/firefox-browser-logo.46d7ba73e194da3d78cb1fca759962c7.svg 2>&1 > /dev/null | grep cache-control
< cache-control: max-age=600,public
```

**Review by sciurus — APPROVED** [review]

**Comment by pmac** [comment]
Seems to be working on stage 

```
$ curl -I https://pocket-static-staging.netlify.app/en/privacy/ 2>&1 | grep cache-control
cache-control: max-age=600,public
```

**Pull request merged**